content/Helpers.js

```diff
@@ -359,16 +359,16 @@ Copper.parseLinkFormat = function(data) {
 		
 		if (elems[2]) {
 		
-			var tokens = elems[2].match(/(;\s*\w+\s*(=\s*(\w+|"([^\\"]*(\\.[^"\\]*)*)"))?)/g);
+			var tokens = elems[2].match(/(;\s*\w+\s*(=\s*([^",;][^,;]+|"([^\\"]*(\\.[^"\\]*)*)"))?)/g);
 		
 			Copper.logEvent(' '+uri+' ('+tokens.length+')');
 		
 			for (var j in tokens) {
 				//Copper.logEvent('  '+tokens[j]+'\n');
-				var keyVal = tokens[j].match(/;\s*([^<"\s;,=]+)\s*(=\s*(([^<"\s;,]+)|"([^"\\]*(\\.[^"\\]*)*)"))?/);
+				var keyVal = tokens[j].match(/;\s*([^<"\s;,=]+)\s*(=\s*(([0-9]+)|([^<"\s;,]+)|"([^"\\]*(\\.[^"\\]*)*)"))?/);

```

Comment:
[0-9]+ should be included in [^<"\s;,]. Not sure what this adds beside another "? check" below... do you have test cases?